main.tf

```diff
@@ -5,30 +5,34 @@
 
 # Create the GKE Cluster
 module "gke" {
-  source                     = "terraform-google-modules/kubernetes-engine/google//modules/beta-private-cluster"
-  version                    = "18.0.0"
-  kubernetes_version         = "1.22.8-gke.202"
-  project_id                 = var.project_id
-  name                       = var.cluster_name
-  regional                   = var.regional
-  region                     = var.region
-  zones                      = var.node_zones
-  network                    = var.network
-  subnetwork                 = var.subnetwork
-  ip_range_pods              = var.ip_range_pods
-  ip_range_services          = var.ip_range_services
-  logging_service            = var.logging_service
-  monitoring_service         = var.monitoring_service
-  maintenance_start_time     = var.maintenance_start_time
-  create_service_account     = var.create_service_account
-  service_account            = var.service_account_email
-  skip_provisioners          = var.skip_provisioners
-  http_load_balancing        = var.http_load_balancing
-  horizontal_pod_autoscaling = var.horizontal_pod_autoscaling
-  network_policy             = var.network_policy
-  enable_private_nodes       = var.enable_private_nodes
-  master_ipv4_cidr_block     = var.master_ipv4_cidr_block
-  remove_default_node_pool   = var.remove_default_node_pool
+  source  = "terraform-google-modules/kubernetes-engine/google//modules/beta-private-cluster"
+  version = "22.1.0"
+  # kubernetes_version         = "1.22.8-gke.202"
```

Comment:
If we aren't using this line anymore, should just delete it rather than leave it in prod code.

Reply:
I think I left it to not forget to ask you what you though about not pinning a version. I was looking at the [release schedule](https://cloud.google.com/kubernetes-engine/docs/release-schedule) and I was a bit confused on what would be share to choose. If we don't pin, we get whatever is lates, and usually that had been okay except for the COS deprecation that we experienced earlier this semester..... but that scramble was annoying... so not sure